community/corepdf/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: rahmanshaber <rahmanshaber@yahoo.com>
+# Maintainer: rahmanshaber <rahmanshaber@yahoo.com>
+pkgname=corepdf
+pkgver=2.6.0
+pkgrel=0
+pkgdesc="A PDF viewer from the CoreApps family."
+arch="all"
+url="https://gitlab.com/cubocore"
+license="LGPL-2.1"
+depends="libcprime qt5-qtwebengine qt5-qtwebchannel qt5-qtsvg"
```

Comment:
first 3 are not required

Reply:
Those are dpends. That app uses pdf.js and it is a web pdf viewer.

Reply:
Alpine automatically depends via checking what shared libraries ELF binaries load.

```
so:libQt5WebChannel.so.5
so:libQt5WebEngineWidgets.so.5
```